cryptography/src/ed25519/scheme.rs

```diff
@@ -438,21 +364,36 @@ mod tests {
         assert!(Ed25519::verify(
             None,
             message,
-            &PublicKey::try_from(public_key.to_vec()).unwrap(),
+            &public_key,
             &computed_signature
         ));
     }
 
     fn parse_private_key(private_key: &str) -> PrivateKey {
-        PrivateKey::try_from(commonware_utils::from_hex_formatted(private_key).unwrap()).unwrap()
+        PrivateKey::decode(
+            commonware_utils::from_hex_formatted(private_key)
+                .unwrap()
+                .as_ref(),
```

Comment:
I suppose its probably overkill to have a version of `decode` that consumes `Vec<u8>` (and for unclear benefit)?